README.md

```diff
@@ -213,8 +215,8 @@ $ gcloud logging read "resource.type=gce_instance AND jsonPayload.SYSLOG_IDENTIF
 
 ### Overview
 
-- [istio-cni.yaml](deployments/kubernetes/install/manifests/istio-cni.yaml)
-   - manifest for deploying `install-cni` container as daemonset
+- [istio-cni.yaml](deployments/kubernetes/install/helm/istio-cni/templates/istio-cni.yaml)
```

Comment:
Please move the templates to istio/installer - that's where all other install templates are maintained.

Reply:
That's out of the scope of this PR.